pages/common/az-config.md

```diff
@@ -0,0 +1,21 @@
+# az config
+
+> Manage Azure CLI configuration.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Print all configurations:
+
+`az config get`
+
+- Print configurations for a section:
+
+`az config get {{section_name}}`
+
+- Set a configuration:
+
+`az config set {{key_value}}`
+
+- Unset a configuration:
+
+`az config unset {{key}}`
```

Comment:
```suggestion
`az config unset {{configuration_name}}`
```

Reply:
Thanks for the suggestions @marchersimon ! I have committed them all 👍